test/smp/regression/saluki/cases/distribution_metrics/lading/lading.yaml

```diff
@@ -0,0 +1,48 @@
+generator:
+  - unix_datagram:
+      seed: [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53,
+             59, 61, 67, 71, 73, 79, 83, 89, 97, 101, 103, 107, 109, 113, 127, 131]
+      path: "/tmp/adp-dsd.sock"
+      block_cache_method: Fixed
+      variant:
+        dogstatsd:
+          contexts:
+            inclusive:
+              min: 1000
+              max: 10000
+          name_length:
+            inclusive:
+              min: 1
+              max: 200
+          tag_length:
+            inclusive:
+              min: 3
+              max: 150
+          tags_per_msg:
+            inclusive:
+              min: 2
+              max: 50
+          multivalue_count:
+            inclusive:
+              min: 2
+              max: 32
+          multivalue_pack_probability: 0.08
+          kind_weights:
+            metric: 100
+            event: 0
+            service_check: 0
+          metric_weights:
+            count: 0
+            gauge: 0
+            timer: 0
+            distribution: 100
+            set: 0
+            histogram: 0
+      bytes_per_second: "100 MiB"
```

Comment:
Seeing 6MiB/s ingress in practice, curiously identical between saluki & ddgo. Might point to a lading problem?